### PR TITLE
suggest new bullets for org-bullets

### PR DESCRIPTION
**Pull request by ralesi**
Also add org-trello mode in case anyone has it re-added back in their `dotspacemacs` config

### CODE REVIEW

**Comment by swaroopch** [comment]
Should we really have org-trello enabled by default? What is the percentage of Org Mode users that also use Trello? :open_mouth:

**Comment by trishume** [comment]
@swaroopch Yah it's an interesting question in terms of Spacemacs philosophy. What is the threshold of including a package in a layer? Should it be it's own layer if only some people want it, won't that lead to bajillions of layers?  What about the fact that all it does is load the package.

Maybe there should be a `.spacemacs` list variable for packages that are just installed and `use-package` loaded without any extra config. For tiny things like this that are not even worth the effort of a private layer. Or maybe private layers are easy enough.

**Comment by CestDiego** [comment]
It would be nice if we could organize them all under a 'org sth sth (layer? variable? wat?) the same for other modes. Althought the lazy loading of it is what actually may be what the Spacemacs philosophy brings to the table.

**Comment by swaroopch** [comment]
@trishume My take: private layers are easy enough, no need to complicate things. For example, I have an org-pomodoro layer - https://github.com/syl20bnr/spacemacs/pull/934

**Comment by ralesi** [comment]
Right now I think the line is still commented, I just added the init files since it was there, but you make are right, we need to periodically review the number of packages we really want to have in the base install.  Anyone have an idea how long it takes to set up from empty elpa directory?  I haven't emptied mine in about 5 months.

**Comment by syl20bnr** [comment]
Thank you, I will wait the next release to merge it, the current release is already too big.

File: spacemacs/packages.el [hunk 1]
```diff
@@ -1987,7 +1987,13 @@ Put (global-hungry-delete-mode) in dotspacemacs/config to enable by default."
 (defun spacemacs/init-org-bullets ()
   (use-package org-bullets
     :defer t
-    :init (add-hook 'org-mode-hook 'org-bullets-mode)))
+    :init (add-hook 'org-mode-hook 'org-bullets-mode)
+    :config (setq org-bullets-bullet-list '("■" "◆" "▲" "▶"))))
```
Comment (syl20bnr) [inline spacemacs/packages.el]: you should put it in `:init` to allow users to easily override the setting in their config.

**Comment by ralesi** [comment]
Updated to current head and declared bullets in init.

**Comment by syl20bnr** [comment]
The default bullets are nicer on this setup, I have to test them on other boxes. It leads me to think that we need a way to propose different types of bullets for the user, there is no silver... bullet that works with all fonts correctly. Since people are now familiar with the circles and flowers (it is spring after all) I don't feel confortable to change them but I see the value to propose different themes.

Do we set a new dotspacemacs variable ? Or maybe better, I plan to add an `org` layer for the next release, we could add a org-bullet-theme variable.

**Comment by syl20bnr** [comment]
Another alternative, the theme feature could be added upstream.

**Comment by trishume** [comment]
+1 for a variable. I personally like all the default bullets except for the flower. Visual preferences are pretty variable person-to-person so it makes sense to have it be a variable.

Plus, there are just so many things to choose from!
screen shot 2015-04-02 at 11 52 05 am

**Comment by syl20bnr** [comment]
Since bullets are more a preference thing I finally took another route by adding your alternative in the documentation of the org layer. See commit 737e2e86e1616c58689247802077fe2185257e02.

Thank you for the PR :+1:

**Pull request closed without merging**